src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -29,56 +29,92 @@
 
 /**
  * <p>
- * Checks the placement of left curly braces.
- * The policy to verify is specified using the {@link LeftCurlyOption} class
- * and the default one being {@link LeftCurlyOption#EOL}.
+ * Checks for the placement of left curly braces (<code>'{'</code>) for code blocks.
```

Comment:
`{@code '{'}` won't work here since it requires balanced curly braces. Same as #5423

Reply:
Unfortunately TC doesn't like this.
> 32: LeftCurlyCheck ...&lt;/code&gt; can be replaced with '{@code ...}' 
37: LeftCurlyCheck ...&lt;/code&gt; can be replaced with '{@code ...}' 
147: option ...&lt;/code&gt; can be replaced with '{@code ...}' 
151: setOption() ...&lt;/code&gt; can be replaced with '{@code ...}' 

Can we use `&lbrace;` or `&#123;` instead inside the `{@code }`?

Reply:
Done. These warnings should be suppressed. And the suppression itself should be ignored.

Reply:
> Can we use `&lbrace;` or `&#123;` instead inside the `{@code }`?

No, the javadoc tool will translate them to &amp;lbrace and &amp;#123;